CONTRIBUTING.rst

```diff
@@ -23,6 +23,22 @@ Contributions
 Contributions are welcome and are greatly appreciated! Every little bit helps,
 and credit will always be given.
 
+Mentoring
```

Comment:
I would change it to 'Get Mentoring Support' since according to the doc structure all sub-topics should start with action verbs:)

Reply:
Updated!